### PR TITLE
fix: unit test failures

### PR DESCRIPTION
**Pull request by itsmostafa**
Test image hook was failing with `ValueError: ('unknown configuration type: pathlist', 'src/ tests/')`

## 🗣 Description ##
Fixed test image by updating config var from the deprecated and removed `python_paths` variable to the new `pythonpath` variable. Also updated pytest hook to python 3.10 to match the project wide python version.

## 💭 Motivation and context ##
test image hook in gh actions is failing. need to fix in order to successfully run gh actions checks on Pull Requests

## 🧪 Testing ##
Testing is being conducted within this PR to trigger github actions

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [pytest]
-addopts = -v -ra --cov
-python_paths = src/
+addopts = -v -ra --runslow
```
Comment (dav3r) [inline pytest.ini]: Why did you get rid of the coverage report (`--cov` flag)?

Comment (itsmostafa) [reply pytest.ini]: Ah sorry, totally forgot to add it back

Comment (itsmostafa) [reply pytest.ini]: @dav3r resolved with [80f6d39](https://github.com/cisagov/domain-manager-api/pull/553/commits/80f6d399a0fb66ab207fa649406f678344eb5149)

File: pytest.ini [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [pytest]
-addopts = -v -ra --cov
-python_paths = src/
+addopts = --cov -v -ra --runslow
```
Comment (dav3r) [inline pytest.ini]: We encourage the use of long flag names when available and might as well alphabetize the flags while we're at it:
```suggestion
addopts = --cov -ra --runslow --verbose
```

**Review by dav3r — CHANGES_REQUESTED** [review]

File: pytest.ini (hunk 2 above)
Comment (itsmostafa) [reply pytest.ini]: good to know. resolved with [d8d9366](https://github.com/cisagov/domain-manager-api/pull/553/commits/d8d93668304b7b3768f38f30d39248e149fff96f)

**Review by dav3r — APPROVED** [review]
Good job getting these tests going again! 👍

**Pull request merged**